tema1/schedule.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My schedule</title>
+  </head>
+
+  <body>
+    <h1>My Schedule</h1>
+    <header>
+      <nav>
+        <p>May 2019<meter value="50" max="100" min="0"></meter></p>
+        <p>
+          <a href="index.html">Home|</a><a href="bio.html">Bio|</a
+          ><a href="schedule.html">Schedule</a>
+        </p>
+      </nav>
+    </header>
+
+    <table border="1" width="70%">
+      <thead>
+        <th></th>
+        <th>Monday</th>
+        <th>Tuesday</th>
+        <th>Wednesday</th>
+        <th>Thursday</th>
+        <th>Friday</th>
+        <th>Saturday</th>
+        <th>Sunday</th>
+      </thead>
+
+      <tr>
+        <td><h2>09-12</h2></td>
+        <td><span>&#10132;</span> Read more</td>
+        <td>
+          HTML<br />
+          <img
+            src="images/imagine pt tabel.jpeg"
```

Comment:
ce as sugera este ca atunci cand folositi imagini, e sa nu folositi spatii in numele fisierelor. Mai indicat este un nume de genul `imagine_pt_tabel`